VERSION

```diff
@@ -1 +1 @@
-0.15.0
+0.15.1
```

Comment:
I would bump this to 0.16.0 since it's a new feature and not a bugfix.

Reply:
I thought about that... Just went with a `.1` because a new version was released last sprint. 
Good to have a 2nd opinion 👍